### PR TITLE
fix(tracer): tracer log deprecation warning

### PR DESCRIPTION
**Pull request by mabdinur**
``dtrace.tracer.log`` is no longer supported. ``logging.getLogger("ddtrace.tracer")`` should be used instead.

## Checklist
- [ ] Added to the correct milestone.
- [ ] Tests provided or description of manual testing performed is included in the code or PR.
- [ ] Library documentation is updated.
- [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).

### CODE REVIEW

**Comment by mabdinur** [comment]
Would it make sense to add a bugfix release note or update the existing tracer.log deprecation note?

**Review by brettlangdon — COMMENTED** [review]
👍🏻

**Comment by brettlangdon** [comment]
> Would it make sense to add a bugfix release note or update the existing tracer.log deprecation note?

Yes, this is a good idea

**Review by Kyle-Verhoog — APPROVED** [review]

File: releasenotes/notes/deprecate-tracer-log-attribute-a7fb4c9c23d5b7b4.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ---
 deprecations:
   - |
-    :py:attr:`ddtrace.Tracer.log` is deprecated. Use :py:data:`ddtrace.tracer.log` instead.
+    :py:attr:`ddtrace.Tracer.log` is deprecated. Use :py:data:`logging.getLogget("ddtrace.tracer")` instead.
```
Comment (majorgreys) [inline releasenotes/notes/deprecate-tracer-log-attribute-a7fb4c9c23d5b7b4.yaml:4]:
```suggestion
    :py:attr:`ddtrace.Tracer.log` is deprecated. Use :py:data:`logging.getLogger("ddtrace.tracer")` instead.
```

File: releasenotes/notes/deprecate-tracer-log-attribute-a7fb4c9c23d5b7b4.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
 ---
 deprecations:
   - |
-    :py:attr:`ddtrace.Tracer.log` is deprecated. Use :py:data:`ddtrace.tracer.log` instead.
+     ``ddtrace.Tracer.log`` is deprecated. Use :py:data:`logging.getLogger("ddtrace.tracer")` instead.
```
Comment (majorgreys) [inline releasenotes/notes/deprecate-tracer-log-attribute-a7fb4c9c23d5b7b4.yaml:4]: Is the extra space a problem?

```suggestion
    ``ddtrace.Tracer.log`` is deprecated. Use :py:data:`logging.getLogger("ddtrace.tracer")` instead.
```

**Review by majorgreys — APPROVED** [review]

**Review by brettlangdon — APPROVED** [review]

**Comment by mabdinur** [comment]
@Mergifyio backport 0.59

**Review by majorgreys — APPROVED** [review]

**Review by majorgreys — APPROVED** [review]

File: releasenotes/notes/deprecate-tracer-log-attribute-a7fb4c9c23d5b7b4.yaml [hunk 3]
```diff
@@ -1,4 +1,4 @@
 ---
 deprecations:
   - |
-    :py:attr:`ddtrace.Tracer.log` is deprecated. Use :py:data:`ddtrace.tracer.log` instead.
+    ``ddtrace.Tracer.log`` is deprecated. Use :py:data:`logging.getLogger("ddtrace.tracer")` instead.
```
Comment (brettlangdon) [inline releasenotes/notes/deprecate-tracer-log-attribute-a7fb4c9c23d5b7b4.yaml:4]: do we want to revert this?

Comment (majorgreys) [reply releasenotes/notes/deprecate-tracer-log-attribute-a7fb4c9c23d5b7b4.yaml:4]: Yes this should be reverted

**Review by majorgreys — APPROVED** [review]

**Review by brettlangdon — APPROVED** [review]

**Pull request merged**